packages/modular-scripts/react-scripts/config/parts/esmViewConfig.js

```diff
@@ -24,34 +27,59 @@ function createConfig({
   };
 }
 
-function createPluginConfig({ isEnvProduction, styleImports }) {
+function createPluginConfig({ isEnvProduction, styleImports, indexPath, env }) {
+  console.log(styleImports);
+  let templateContent;
+
+  // If we have an index template in the project, let's use it
+  if (indexPath) {
+    templateContent = fs.readFileSync(indexPath, 'utf8');
+
+    // Style (global) imports are excluded from the build, so they don't get injected. We have to manually inject them in the index template.
+    if (styleImports.length) {
+      const page = parse5.parse(templateContent);
+      const html = page.childNodes.find((node) => node.nodeName === 'html');
+      const head = html.childNodes.find((node) => node.nodeName === 'head');
+
+      styleImports.forEach((importUrl) =>
+        head.childNodes.push(
+          ...parse5.parseFragment(
+            `<link rel="stylesheet" href="${importUrl}" />`,
+          ).childNodes,
+        ),
+      );
+      templateContent = parse5.serialize(page);
+    }
+  } else {
+    // Standard minimal template to start an esm-view
+    templateContent = dedent(`
+      <!DOCTYPE html>
+        <head>
+          ${styleImports?.map(
+            (importUrl) =>
+              `<link rel="stylesheet" href="${importUrl}"></script>`,
+          )}
+        </head>
+        <html>
```

Comment:
Do you have a duplicate `html` tag here? Looks like it

Reply:
Not new to your code but looks like an existing issue. Presumably the browser is automatically fixing this but it looks like invalid html to me (the html tag should wrap everything except the doctype)

Reply:
Looks like a bug - moved